src/Components/Components/src/HotReload/HotReloadManager.cs

```diff
@@ -20,6 +20,6 @@ public static void DeltaApplied()
             OnDeltaApplied?.Invoke();
         }
 
-        public static void OnAfterUpdate(Type[]? _) => OnDeltaApplied?.Invoke();
+        public static void AfterUpdate(Type[]? _) => OnDeltaApplied?.Invoke();
```

Comment:
The method being looked up is `AfterUpdate`, not `OnAfterUpdate`.  I thought I tested this when I made the change, but I clearly got this wrong.